src/openforms/forms/migrations/0042_explanation_template_data_migration.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.14 on 2022-08-26 15:29
+from django.db import migrations
+
+
+def append_explanation_template_message(apps, schema_editor):
+    from openforms.authentication.registry import register as auth_register
+    from django.utils.translation import gettext_lazy as _
```

Comment:
why is this a local import?

You can also use the non-lazy variant, we aren't defining any translations strings at the module level.

Reply:
I want to separate concerns between migration and reverse migration.

Can you explain the advantages of using the non-lazy variant in this particular case?

Reply:
> I want to separate concerns between migration and reverse migration.

Imports in python always go to the top of the module, unless you are dealing with circular imports.

For the same reason, the `register` import should probably also just go to the top of the module.

> Can you explain the advantages of using the non-lazy variant in this particular case?

It doesn't output lazy strings, but actual translated strings. The lazy variant in Django only exists because often translations need to be defined at module level (so loaded/evaluated at import-time) and be translated at runtime.

Reply:
> It doesn't output lazy strings, but actual translated strings. The lazy variant in Django only exists because often translations need to be defined at module level (so loaded/evaluated at import-time) and be translated at runtime.

What advantage are we gaining from outputting translated strings over lazy strings in this migration?

Reply:
This is a Django convention per the docs: https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#standard-translation

You should use `gettext` unless there's concrete reason to use the lazy variant. The lazy variant messes a lot with type hints where you declare a `str` typehint, so it is preferential. But the main reason is the Django convention to use the non-lazy call.